panel/chat/feed.py

```diff
@@ -337,6 +337,13 @@ def _update_placeholder(self):
             **self.placeholder_params
         )
 
+    @param.depends("loading", watch=True, on_init=True)
```

Comment:
How does this interact with the usual loading spinner?

Reply:
Do you mean something like `pn.Column("Hello", loading=True)`?

<img width="88" alt="image" src="https://github.com/user-attachments/assets/a36da898-e58b-43da-b137-435792704d6c">

I don't think anything gets overlaid on ChatInterface besides the individual buttons
`pn.chat.ChatInterface("Hello", loading=True).show()`

<img width="1303" alt="image" src="https://github.com/user-attachments/assets/147686ec-846c-4339-9bdb-d42205997307">